app/lib/actions/katello/agent_action.rb

```diff
@@ -14,8 +14,9 @@ def agent_action_type
 
       def plan(host, options)
         action_subject(host, :hostname => host.name, :content => options[:content])
-
-        dispatch_history_id = options.dig(:dispatch_histories, host.id.to_s) || ::Katello::Agent::Dispatcher.create_histories(
+        dispatch_history_id = options.dig(:dispatch_histories, host.id.to_s) ||
```

Comment:
Code looks fine, will start testing. One comment here, do you think we should add a comment what this is doing like you did in `bulk_agent_action.rb`